src/Serializer/ItemNormalizer.php

```diff
@@ -37,11 +39,17 @@ public function normalize($object, $format = null, array $context = [])
 
     /**
      * {@inheritdoc}
+     *
+     * @throws InvalidArgumentException
      */
     public function denormalize($data, $class, $format = null, array $context = [])
     {
         // Avoid issues with proxies if we populated the object
         if (isset($data['id']) && !isset($context['object_to_populate'])) {
+            if (isset($context['allow_update']) && true !== $context['allow_update']) {
```

Comment:
Problems are very unlikely. Our options are mostly for our normalizers. To have a conflict we have to:

* Create a new normalizer (or import it from a bundle)
* Have a collision between a key of this normalizer's context and one defined (we can document all available keys to mitigate this)
* Have a case where the third-party normalizer is called by our normalizer (very unlikely)

IMO it's not a big deal.